src/components/AcceptOrRejectAnalytics.tsx

```diff
@@ -31,17 +31,14 @@ export function AcceptOrRejectAnalytics({
           </p>
         </div>
         <div>
-          <p className="paragraph">
-            <div
-              role="paragraph"
```

Comment:
curious what prompted these changes? just extra wrappers?

Reply:
(@eligrey  also had this question but responding here)

this is actually improper html spec! i was running these through the prettier html parser and was getting errors on it. i forget the exact pertinent section but i think its something about only having inline tags inside of p tags.

regardless, this was poor html to begin with bc of the extra wrapper thing you mentioned, and -also- (to address elis question) worse for accessibility! the role="paragraph" was necessary because we were using divs that are otherwise opaque to screen readers/accessibility trees. if we remove it, we're simplifying the dom and also improving screen reader/accessibility tree parsing